src/config.py

```diff
@@ -96,6 +108,14 @@ def jaas_users(self) -> List[str]:
             jaas_users.append(f'user_{username}="{password}"')
         return jaas_users
 
+    @property
+    def metrics_exporter_config(self) -> list[str]:
+        """Necessary config options for enabling built-in Prometheus metrics."""
+        return [
+            "metricsProvider.className=org.apache.zookeeper.metrics.prometheus.PrometheusMetricsProvider",
+            "metricsProvider.httpPort=7000",
```

Comment:
Change this to use `METRICS_PROVIDER_PORT` literal

Reply:
Duh, thank you!